packages/traefik-config-scripts/spec.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
actually spec.yml from traefik have same "signature" in begin.
Actually I copy it, and remove unrelated as well as remove unrelated form original spec